webcompat/templates/issue-wizard-form.html

```diff
@@ -68,7 +68,11 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
               {% endfor %}
             {% endif %}
           </div>
-          {{form.problem_category(autocomplete='off')}}
```

Comment:
(note: this `autocomplete=off` was getting set on the parent `<ul>` element, so it was basically useless.)